planet-express/PlanetExpress/src/planet/PlanetExpress.java

```diff
@@ -0,0 +1,110 @@
+package planet;
+
+import java.util.Scanner;
+
+public class PlanetExpress {
+	int thirst;
+	int work;
+	int receipts;
+	int horde;
+	int hunger;
+	boolean stable;
+	
+	PlanetExpress() {
+		thirst = 50;
+		work = 50;
+		receipts = 50;
+		horde = 50;
+		hunger = 50;
+		stable = true;
+		
+	}
+
+	public static void main(String[] args) {
+		PlanetExpress p = new PlanetExpress();
+		System.out.println("Welcome to Planet Express!");
+		while (p.stable) {
+			Scanner reader = new Scanner(System.in);  // Reading from System.in
+			System.out.println("Type drink, deliver, steal, eat, or account to continue playing.");
+			String input = reader.next();
+			while (!input.toLowerCase().equals("drink") && !input.toLowerCase().equals("deliver") && !input.toLowerCase().equals("steal")
```

Comment:
It seems like you should be doing the lower case operation only once rather than in each of these equivalence checks